homeassistant/components/livisi/config_flow.py

```diff
@@ -71,6 +91,13 @@ async def create_entity(
         self, user_input: dict[str, str], controller_info: dict[str, Any]
     ) -> FlowResult:
         """Create LIVISI entity."""
+        if self.reauth_entry:
+            self.hass.config_entries.async_update_entry(
+                self.reauth_entry, data=controller_info
```

Comment:
This feels wrong.
Should data not simply be updated with the password?

Reply:
You are right, I changed this, thank you.